src/posts/2023-02-20-0xmonaco-battle-of-titans/index.md

```diff
@@ -0,0 +1,148 @@
+---
+path: /posts/0xmonaco-battle-of-titans
+title: "0xMonaco - Battle of Titans"
+author: miguel-palhas
+date: 2023-02-20
+cover:
+seoImage:
+intro: ""
+---
+
+[0xmonaco]: https://www.google.com/url?q=https://mirror.xyz/matchboxdao.eth/-xVvvo36y-LHvrW99heb4VJ-xAOWQ-BQAx6MCwHLWnk&sa=D&source=docs&ust=1676917568372077&usg=AOvVaw2_oBhlTbilNdAPWV2uKQ8T
+[gda]: https://www.paradigm.xyz/2022/04/gda
+[resende]: https://github.com/ZepedroResende
+[writeups-thread]: https://twitter.com/abdulsamijay/status/1561821616778141696
+[brock]: https://www.google.com/url?q=https://hackmd.io/@onemanbandplus2/S1Ez1Ulys&sa=D&source=docs&ust=1676917940239184&usg=AOvVaw2kPWeNf1MEDNhGh7Lkeksc
+[zellic]: https://twitter.com/zellic_io/status/1620844495226179586
+[resende]: https://twitter.com/Resende_666
+[elo]: https://en.wikipedia.org/wiki/Elo_rating_system
+[foundry]: https://book.getfoundry.sh/
+[robocode]: https://robocode.sourceforge.io/
+[me]: https://twitter.com/naps62
+
+On January 5th, we took notice of MatchboxDAO’s event, [0xMonaco][0xmonaco].
+
+![](car.gif)
+
+0xMonaco is a pvp racing game, originally designed by Paradigm for their annual CTF contest. The event is a Mario Kart-inspired racing game, where cars are controlled by a smart contract written by each competing team.
+
+The main mechanics are built around resource management, since each car needs to continuously make decision about resources to purchase (throw shells at other players, or even speed itself to advance in the race). Resource costs are controlled by a [GDA (gradual dutch auction)][gda], with prices floating up/down exponentially based on short-term demand.
```

Comment:
"The main mechanics are built around resource management, since each car needs to continuously make _decisions_"